examples/platform/silabs/MatterConfig.cpp

```diff
@@ -46,10 +46,10 @@
 #endif
 
 // TODO: We shouldn't need any platform specific includes in this file
-#if defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1
+#if ((defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1) || defined(EXP_BOARD))
 #include <platform/silabs/SiWx917/SiWxPlatformInterface.h>
```

Comment:
#include <platform/silabs/SiWx917/SiWxPlatformInterface.h> im not sure this is valid for ncp. please confirm

Reply:
Added include only for SoC